src/pages/admin/reports/components/funds-expenditures-section/components/funds-expenditures-table/FundsExpendituresTable.module.scss

```
@@ -2,6 +2,29 @@
 @use '@/assets/sass/mixins/typography.mixins' as type;
 @use '@/assets/sass/variables/z-index';
 
+// Local color aliases (use `-color` suffix for background-related variables)
+$bg-light-blue-color: c.$light-blue-25;
+$bg-light-blue-50-color: c.$light-blue-50;
+$bg-light-blue-100-color: c.$light-blue-100;
+$bg-light-blue-200-color: c.$light-blue-200;
+$bg-white-color: c.$white;
+$bg-blue-500-color: c.$blue-500;
+
+$bg-grey-300-color: c.$grey-300;
```

Stylelint (17.9.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)